includes/class-newspack-ads-model.php

```diff
@@ -221,37 +226,33 @@ public static function delete_ad_unit( $id ) {
 	 * Update/create the header code for a service.
 	 *
 	 * @param string $service The service.
-	 * @param string $header_code The code.
+	 * @param string $network_code The code.
 	 */
-	public static function set_header_code( $service, $header_code ) {
-		if ( ! current_user_can( 'unfiltered_html' ) ) {
-			return false;
-		}
-		$id = self::NEWSPACK_ADS_SERVICE_PREFIX . $service . self::NEWSPACK_ADS_HEADER_CODE_SUFFIX;
-		update_option( self::NEWSPACK_ADS_SERVICE_PREFIX . $service . self::NEWSPACK_ADS_HEADER_CODE_SUFFIX, $header_code );
+	public static function set_network_code( $service, $network_code ) {
+		$id = self::NEWSPACK_ADS_SERVICE_PREFIX . $service . self::NEWSPACK_ADS_NETWORK_CODE_SUFFIX;
+		update_option( self::NEWSPACK_ADS_SERVICE_PREFIX . $service . self::NEWSPACK_ADS_NETWORK_CODE_SUFFIX, sanitize_text_field( $network_code ) );
 		return true;
 	}
 
 	/**
 	 * Retrieve the header code for a service.
 	 *
 	 * @param string $service The service.
-	 * @return string $header_code The code.
+	 * @return string $network_code The code.
 	 */
-	public static function get_header_code( $service ) {
-		return get_option( self::NEWSPACK_ADS_SERVICE_PREFIX . $service . self::NEWSPACK_ADS_HEADER_CODE_SUFFIX, '' );
+	public static function get_network_code( $service ) {
+		return get_option( self::NEWSPACK_ADS_SERVICE_PREFIX . $service . self::NEWSPACK_ADS_NETWORK_CODE_SUFFIX, '' );
```

Comment:
Sanitize this option before returning. People might extend this plugin in a child theme or something.

Reply:
For knowledge-sharing purposes, I'll expand on this. It's common for people to think that because the data is sanitized on the way in from this plugin, it will remain sanitized on the way out. This is not a safe assumption to make since all plugins and themes have access to the WP DB. The data could get modified by some other code after it's been saved, so we have to always assume everything straight from the DB is dangerous.

Reply:
https://github.com/Automattic/newspack-ads/pull/15/commits/d116e92900961962b98433db0ec5a9a3b2b1ea8e